{{cookiecutter.project_name}}/src/{{cookiecutter.package_name}}/pp/basic.py

```diff
@@ -1,5 +1,14 @@
+from collections.abc import Callable, Iterable
+from typing import Any, TypeVar
+
+import numpy as np
 from anndata import AnnData
 
+MuData = TypeVar("MuData")
```

Comment:
Why is TypeVar required here? Can't you use MuData/AnnData directly? 

or

```
if TYPE_CHECKING:
   from mudata import MuData
   from spatialdata import SpatialData
```

if you are worried about additional imports?

Reply:
Unless I'm stupid, we'd then also require some patching for sphinx (mock imports or nitpick ignores etc). I'd like to avoid that for MuData and SpatialData because people might not notice that we did so. I think it's easier if we keep it as I had it.

Reply:
ok, fair enough